sgl-kernel/tests/test_moe_align.py

```diff
@@ -144,7 +144,7 @@ def moe_align_block_size_triton(
             [32, 64, 128, 256],  # block_size
             [1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 2048, 4096],  # num_tokens
             [1, 2, 4, 8, 16, 32, 64],  # topk
```

Comment:
nit: not run the code, but if I understand correctly, topk now is 1 larger, so maybe we can put that value here as well

Reply:
done